webcompat/templates/issue/issue-labels.jst

```diff
@@ -1,14 +1,14 @@
-<script type="text/template" id="issue-labels-tmpl">
+<script type="text/template">
   <header class="wc-Issue-labelEditor">
     <div class="wc-Issue-labelEditor-title">
       Labels
     </div>
-    {% if session.user_id and session.avatar_url %}
+    <% if ($("body").data("username")) { %>
```

Comment:
Note: the python backend will return a 403 if a user isn't authenticated when trying to modify labels, so I feel pretty fine with moving this to the front-end.